crates/pop-cli/src/commands/new/parachain.rs

```diff
@@ -216,7 +216,7 @@ async fn generate_parachain_from_template(
 	// add next steps
 	let mut next_steps = vec![
 		format!("cd into \"{name_template}\" and enjoy hacking! 🚀"),
-		"Use `pop build` to build your parachain.".into(),
+		"Use `pop build --release` to build your parachain.".into(),
```

Comment:
I made this change because it's common in the Zombienet configuration file to reference `/target/release/`.  See https://github.com/OpenZeppelin/polkadot-runtime-templates/blob/main/generic-template/zombienet-config/devnet.toml#L21 or https://github.com/r0gue-io/pop-cli/blob/main/tests/networks/template.toml#L16.
Without running `pop build --release`, the path needs to be manually updated in the Zombienet file for things to work correctly.